### ISSUE TITLE
unhelpful crash message when file in index is empty

### ISSUE THREAD

**Issue by flashton2003** [will-rowe/groot#18]
I'm trying to build a groot index and it returned this error

```panic: multi: index out of range

goroutine 39 [running]:
github.com/biogo/biogo/seq/multi.(*Multi).Column(0xc00001d780, 0x0, 0x1, 0x0, 0x0, 0x0)
	/opt/conda/conda-bld/groot_1553596083523/work/src/github.com/biogo/biogo/seq/multi/multi.go:250 +0x691
github.com/will-rowe/gfa.getNodes(0xc00001d780, 0x1, 0x0, 0x0)
	/opt/conda/conda-bld/groot_1553596083523/work/src/github.com/will-rowe/gfa/msa.go:150 +0xa42
github.com/will-rowe/gfa.MSA2GFA(0xc00001d780, 0xaf3010, 0xc0000240d0, 0x0)
	/opt/conda/conda-bld/groot_1553596083523/work/src/github.com/will-rowe/gfa/msa.go:46 +0x117
github.com/will-rowe/groot/cmd.runIndex.func1(0xc0000240d0, 0xc0000c4240, 0x12, 0xc00001d780)
	/opt/conda/conda-bld/groot_1553596083523/work/src/github.com/will-rowe/groot/cmd/index.go:168 +0x59
created by github.com/will-rowe/groot/cmd.runIndex
	/opt/conda/conda-bld/groot_1553596083523/work/src/github.com/will-rowe/groot/cmd/index.go:165 +0x55f```

Turns out I had an empty file in my input directory, removing it makes it work. Maybe kind of niche (what kind of dumbass includes an empty file in their index directory, right?!), but might be good for groot to do this as a sanity check?

**Comment by will-rowe** [comment]
Oops - thanks for pointing that out. I will add that check for the next release.

**Comment by will-rowe** [comment]
giving groot some much needed love - this is finally fixed with f507df489fd1d0d38b159563c9afeb3328012d9d
